qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/ModelIT.java

```diff
@@ -55,7 +53,7 @@ public class ModelIT extends AbstractRestartUpgradeTestCase {
     private static int DOC_ID_TEST_MODEL_INDEX = 0;
     private static int DOC_ID_TEST_MODEL_INDEX_DEFAULT = 0;
     private static final int DELAY_MILLI_SEC = 1000;
-    private static final int EXP_NUM_OF_MODELS = 3;
+    private static final int EXP_NUM_OF_MODELS = 2;
```

Comment:
did you find out why expected num of models has changed from 3 to 2?

Reply:
Yes, it's because there was another model created in the testDeleteTrainingModel() method that was deleted in this PR.  The remaining tests only create two models.